dspace-api/src/test/java/org/dspace/external/provider/impl/OrcidPublicationDataProviderIT.java

```diff
@@ -0,0 +1,387 @@
+import java.util.function.Predicate;
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.Unmarshaller;
+
+import org.apache.commons.codec.binary.StringUtils;
+import org.dspace.AbstractIntegrationTestWithDatabase;
+import org.dspace.app.orcid.client.OrcidClient;
+import org.dspace.app.orcid.client.OrcidConfiguration;
+import org.dspace.app.orcid.model.OrcidTokenResponseDTO;
+import org.dspace.builder.CollectionBuilder;
+import org.dspace.builder.CommunityBuilder;
+import org.dspace.builder.ItemBuilder;
+import org.dspace.content.Collection;
+import org.dspace.content.MetadataFieldName;
+import org.dspace.content.dto.MetadataValueDTO;
+import org.dspace.external.model.ExternalDataObject;
+import org.dspace.utils.DSpace;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.orcid.jaxb.model.v3.release.record.Work;
+import org.orcid.jaxb.model.v3.release.record.WorkBulk;
+import org.orcid.jaxb.model.v3.release.record.summary.Works;
+
+/**
+ * Integration tests for {@link OrcidPublicationDataProvider}.
+ *
+ * @author Luca Giamminonni (luca.giamminonni at 4science.it)
+ *
+ */
+public class OrcidPublicationDataProviderIT extends AbstractIntegrationTestWithDatabase {
+
+    private static final String BASE_XML_DIR_PATH = "./target/testing/dspace/assetstore/orcid-works/";
+
+    private static final String ACCESS_TOKEN = "32c83ccb-c6d5-4981-b6ea-6a34a36de8ab";
+
+    private static final String ORCID = "0000-1111-2222-3333";
+
+    private OrcidPublicationDataProvider dataProvider;
+
+    private OrcidConfiguration orcidConfiguration;
+
+    private OrcidClient orcidClient;
+
+    private OrcidClient orcidClientMock;
+
+    private String originalClientId;
+
+    private Collection persons;
+
+    @Before
+    public void setup() throws Exception {
+        context.turnOffAuthorisationSystem();
+
+        parentCommunity = CommunityBuilder.createCommunity(context)
+            .withName("Parent Community")
+            .build();
+
+        persons = CollectionBuilder.createCollection(context, parentCommunity)
+            .withEntityType("Person")
+            .withName("Profiles")
+            .build();
+
+        context.restoreAuthSystemState();
+
+        dataProvider = new DSpace().getServiceManager()
+            .getServiceByName("orcidPublicationDataProvider", OrcidPublicationDataProvider.class);
+
+        orcidConfiguration = new DSpace().getServiceManager()
+            .getServiceByName("org.dspace.app.orcid.client.OrcidConfiguration", OrcidConfiguration.class);
+
+        orcidClientMock = mock(OrcidClient.class);
+        orcidClient = dataProvider.getOrcidClient();
+
+        dataProvider.setReadPublicAccessToken(null);
+        dataProvider.setOrcidClient(orcidClientMock);
+
+        originalClientId = orcidConfiguration.getClientId();
+        orcidConfiguration.setClientId("DSPACE-CRIS-CLIENT-ID");
+        orcidConfiguration.setClientSecret("DSPACE-CRIS-CLIENT-SECRET");
```

Comment:
Please change the DSPACE-CRIS example settings here to just say DSPACE (or something else)